src/platform/nrfconnect/wifi/WiFiManager.cpp

```diff
@@ -469,6 +480,19 @@ void WiFiManager::DisconnectHandler(Platform::UniquePtr<uint8_t>)
     });
 }
 
+void WiFiManager::IPv6PrefixAddHandler(const void * data)
+{
+    // This is needed to send mDNS queries containing new IPv6 prefix
+    chip::DeviceLayer::ChipDeviceEvent event;
```

Comment:
nit: probably we can do 
```suggestion
   ChipDeviceEvent event {.Type = DeviceEventType::kDnssdRestartNeeded};
```
(similarly for kDnssdInitialized to be consistent)
And I would consider using namespace ::chip::DeviceLayer to have less namespaces in place.

Reply:
The code is already inside `DeviceLayer` so not even "using namespace" is needed

Reply:
Ahh I mistakenly assumed it's only scoped to ToScanResponse.